Lab2/ComputerStore.BusinessLogicLayer/Models/Buyer.cs

```diff
@@ -1,13 +1,31 @@
-﻿namespace ComputerStore.BusinessLogicLayer.Models
+﻿using System.ComponentModel.DataAnnotations;
+using ComputerStore.BusinessLogicLayer.Validation.RegexStorage;
+
+namespace ComputerStore.BusinessLogicLayer.Models
 {
     public class Buyer
     {
+        [Required]
```

Comment:
I guess it would be better to create view models for all entities and move all validation attributes to that.